### ISSUE TITLE
[Bug]: ADS Driver has issues writing Structs

### ISSUE THREAD

**Issue by chrisdutz** [apache/plc4x#1391]
### What happened?

When running the manual tests, it turns out that there seem to be issues with writing Structs.

### Version

0.11.0-SNAPSHOT

### Programming Languages

- [X] plc4j
- [ ] plc4go
- [ ] plc4c
- [ ] plc4net

### Protocols

- [ ] AB-Ethernet
- [X] ADS /AMS
- [ ] BACnet/IP
- [ ] CANopen
- [ ] DeltaV
- [ ] DF1
- [ ] EtherNet/IP
- [ ] Firmata
- [ ] KNXnet/IP
- [ ] Modbus
- [ ] OPC-UA
- [ ] S7

**Comment by chrisdutz** [comment]
I think I have found the issue:
- When reading a Struct/UDT, we get a chunk of bytes back from the server. The Tag information defines which element is located where and how long it is. 
- Some times the start of the next tag leaves a gap and when parsing we simply skip those bytes
- Currently when writing we don't fill these gaps (This is what needs to be changed)

I have contacted my guy at Beckhoff, asking him if it's safe to simply fill these gaps with zero-bytes, or if we need to fetch the chunk we want to update first and then to update it, leaving the skipped bytes unchanged.
